crane-cloud/frontend
# Fix project card border

# Description

This PR fixes the issue of elements shifting while hovering over a project card, caused by an added border on hover, that was not initially present. It also adds CSS modules to the component.

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID

https://trello.com/c/hNswSbLu

## How Can This Be Tested?

Pull this branch and run it locally, go to the projects list and hover over different project cards. The elements should have stopped shifting.


## Checklist:


# Screenshots
## Before
![Before border](https://user-images.githubusercontent.com/41950223/129510149-e4b9164c-ef55-44ec-b645-7f7c20cb6529.gif)

## After
![After border](https://user-images.githubusercontent.com/41950223/129510151-f0fba083-f184-4987-86f7-eee3e34d61a8.gif)
